include/report.h

```diff
@@ -60,6 +60,8 @@ struct sr_report
     struct sr_report_custom_entry *auth_entries;
 
     uint32_t serial;
+
+    bool unpackaged;
```

Comment:
Well, I don't understand why you need this. If there is no packaging information, then consider the report unpackaged?

Reply:
@jfilak Having no packaging information in a dump dir doesn't necessarily mean a crashed binary is unpackaged. An unpackaged element is created in abrt-action-dave-package-data only if abrt cannot obtain rpms information. Basically, with this approach, we are able to distinguish either the binary is unpackaged or there is some bug in a code.

Reply:
Then please, clearly state the field is here for debugging purposes. However, I still believe you don't need it. IMHO it just introduces an unnecessary redundancy. The more files you need to care about, the more problems you have.